src/CarbonAware.DataSources/CarbonAware.DataSources.ElectricityMaps/test/Configuration/ServiceCollectionExtensionTests.cs

```diff
@@ -43,7 +46,7 @@ public void ClientProxyTest_With_Missing_ProxyURL_ThrowsException()
 
     [TestCase(true, TestName = "ClientProxyTest, successful: denotes adding ElectricityMaps data sources using proxy.")]
     [TestCase(false, TestName = "ClientProxyTest, successful: denotes adding ElectricityMaps data sources without using proxy.")]
-    public void ClientProxyTest_AddsServices(bool withProxyUrl)
+    public void ClientProxyTest_AddsDataSource(bool withProxyUrl)
```

Comment:
It feels like the method name should be something different, like: `Client_Proxy_Configuration()`, since testing with a proxy in the middle, it requires to perform a remote call by the client.